### ISSUE TITLE
The Timeline - Advanced multiple targets example in docs does not work

### ISSUE THREAD

**Issue by holyjak** [bitworking/react-gsap#65]
I have copied the example from https://bitworking.github.io/react-gsap/src-components-timeline#advanced-multiple-targets into a vanilla create-react-app app and it does not work, namely no animation shows up. When I click "play" in the controls, the time progresses but nothing shows up. I expect the three words to show up. There is no error or warning in the console.

The code is here: https://github.com/holyjak/tmp-react-playground/blob/main/src/App.js#L5-L38

I guess it must be me doing something wrong, but what? Any help most appreciated!

gsap 3.10.4, react-gsap 3.2.1, react 18.1.0

**Comment by bitworking** [comment]
I guess the problem is React 18. I'll try to find time to fix it.

**Comment by holyjak** [comment]
You are right! I thought I also downgraded to react 17 but I did not. I tried that node (rm -rf node_modeules FTW!) and indeed it works with React 17.

**Comment by zanedev** [comment]
Same for me downgrading to react 17 fixed it.
